app/lib/rocketchat.js

```diff
@@ -509,21 +510,25 @@ const RocketChat = {
 		}
 		return Promise.resolve(result);
 	},
-	async getPermalink(message) {
+	async getPermalink(obj, type) {
```

Comment:
It's not consistent.
We're planning to add TypeScript and this is the kind of methods that would require an `any` type on `obj` param.
IMO this method should be split in two: on for message and other for channel.
What do you think?

Reply:
nice!